test/each.scss

```diff
@@ -0,0 +1,17 @@
+$test1: (
+    desktop: 200px,
+    tablet: 350px,
+    mobile: 400px,
```

Comment:
@alexander-schranz remove comma in last line.

Reply:
@c00n84 sure? for git it is better and in php we also have this comma

/cc @thomasduenser @dominikmatt @sabinebaer @rs2487

Reply:
@alexander-schranz @c00n84 Hard decision. It feels strange in SCSS, but I think it's a good idea to handle it the same way as in PHP as it's easier to review.